flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/expressions/TemporalTypesTest.scala

```diff
@@ -562,6 +563,13 @@ class TemporalTypesTest extends ExpressionTestBase {
       "2018-03-14 01:02:03.123456")
   }
 
+  @Test
+  def testAddMonths(): Unit = {
+    testSqlApi("add_months('2016-08-31', 1)", "2016-09-30")
```

Comment:
It would be great to have negative tests as well like with wrong number of arguments, wrong types